src/twisted/internet/cfreactor.py

```diff
@@ -401,24 +500,28 @@ def _scheduleSimulate(self, force=False):
 
         @type force: C{bool}
         """
-        if self._currentSimulator is not None:
-            CFRunLoopTimerInvalidate(self._currentSimulator)
+        self._stopSimulating()
+        if not self._started:
+            # If the reactor is not running (e.g. we are scheduling callLater
+            # calls before starting the reactor) we should not be scheduling
+            # CFRunLoopTimers against the global CFRunLoop.
+            return
+
+        timeout = 0.0 if force else self.timeout()
+        if timeout is None:
```

Comment:
I know that this was already in trunk.
I is not clear what is the semantic of `timeout = None`

Reply:
This is a common feature of pretty much every reactor; it just means "there are no scheduled calls". 

This ought to be documented in `twisted.internet.base.ReactorBase.timeout` but that's a separate change :)